src/otx/algorithms/visual_prompting/adapters/pytorch_lightning/models/visual_prompters/segment_anything.py

```diff
@@ -159,8 +159,14 @@ def replace_state_dict_keys(state_dict, revise_keys):
             # state_dict from args.load_from
             state_dict = replace_state_dict_keys(state_dict, revise_keys)
             self.load_state_dict(state_dict)
+        elif self.config.model.checkpoint == "pretrained":
```

Comment:
How about to write a checkpoint path  in each model config file statically. I think it's better than patching.

Reply:
I just worried some problems when applying configs in `inference` and loading checkpoint in here.
But, I think it doesn't matter but changing them seems to take time.
If it is not critical, can I change them in other PR?

Reply:
yes, then let's change it in next PR. It would be better to reduce dynamic patching, I think.